config.yml

```diff
@@ -0,0 +1,19 @@
+otp:
+  dht:
+    interval: 360
```

Comment:
I guess this is a leftover, right?

Reply:
yeah, there are also testing script, and other leftovers for demo purposes out of the box
that ofc will be cleaned if the main direction of that pr resolved, so further cleaning is upcoming